### ISSUE TITLE
Request: MDN Accessibility

### ISSUE THREAD

**Issue by silvenon** [Kapeli/Dash-User-Contributions#2522]
Hey, @Kapeli, as I understand you maintain some MDN docsets like CSS, HTML, and JS. Do you think you could create one for the [MDN Accessibility reference](https://developer.mozilla.org/en-US/docs/Web/Accessibility) as well? If not, do you have source code for how you generate other MDN docsets so we can use some of that logic to create the one for accessibility?

**Comment by Kapeli** [comment]
I haven't seen much demand for a MDN Accessibility docset. If more users ask, I'll add it.

The scripts I use to generate the CSS/HTML/JS docsets would be of no use, as I only index APIs for those docsets, while the docs you link to seem to contain only guides/tutorials.

**Comment by knowler** [comment]
I would love this, particularly the [ARIA reference](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques).

**Comment by mvshmakov** [comment]
+ for the accessibility.

**Comment by macx** [comment]
I agree, accessibility is not only nice to have in the EU, it's required. To have the ARIA reference on my fingertips would speed up my work.
